plugins/catalog-backend/src/database/DefaultProcessingDatabase.ts

```diff
@@ -57,6 +58,7 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
       database: Knex;
       logger: Logger;
       refreshInterval: ProcessingIntervalFunction;
+      conflictEventBroker?: EventBroker;
```

Comment:
Should we call this just eventBroker and use the topic to indicate the conflict?

Reply:
Should we assume this is already set and default it in the catalog builder?

Reply:
Yes good idea